omnia_timeseries/http_client.py

```diff
@@ -5,17 +5,22 @@
 from omnia_timeseries.helpers import retry
 from requests.models import Response
 import json
+from importlib import metadata
+import sys
 
 logger = logging.getLogger(__name__)
+version = metadata.version("omnia-timeseries-api")
+
 
 class TimeseriesRequestFailedException(Exception):
-    def __init__(self, response:Response) -> None:
+    def __init__(self, response: Response) -> None:
         error = json.loads(response.text)
         self._status_code = response.status_code
         self._reason = response.reason
         self._message = error["message"]
         self._trace_id = error["traceId"]
-        super().__init__(f"Status code: {self._status_code}, Reason: {self._reason}, Message: {self._message},  Trace ID: {self._trace_id}")
+        super().__init__(
+            f"Status code: {self._status_code}, Reason: {self._reason}, Message: {self._message},  Trace ID: {self._trace_id}")
```

Comment:
Why space?

Reply:
Which one? This is just reformatted code, according to python conventions. Any whitespace in the string is just at is has been.

Reply:
@eaidland?

Reply:
Ok it's prob just the formatter

Reply:
There was an extra space before `Trace ID` though, so removed it